tdms/include/cell_coordinate.h

```diff
@@ -25,4 +37,15 @@ class CellCoordinate {
   int i() { return cell_i; };   //< fetch i
   int j() { return cell_j; };   //< fetch j
   int k() { return cell_k; };   //< fetch k
+
+  /**
+   * @brief Copies the values of the cell indices into the addresses provided
+   *
+   * @param[out] i,j,k Address to place the value of cell_{i,j,k} into (respectively)
+   */
+  void unpack(int *i, int *j, int *k) {
+    *i = cell_i;
+    *j = cell_j;
+    *k = cell_k;
+  }
```

Comment:
I'd be a bit wary of [speculative generality](https://refactoring.guru/smells/speculative-generality) 👻 
What method(s) are we _actually_ going to want in the future that isn't just simple public-members `struct` behaviour?

Reply:
You can even add a constructor for the `struct` if you don't like `MyStruct flibble = {foo, bar};` ...
```c++
#include <iostream>

struct CellCoordinate {
  int i, j, k;
  CellCoordinate(int a, int b, int c): i(a), j(b), k(c) {}
};

std::ostream& operator<<(std::ostream& os, const CellCoordinate& c) {
    os << c.i << ',' << c.j << ',' << c.k;
    return os;
}

int main() {
    // standard way:
    //CellCoordinate c = {1,2,3};
    // also works:
    CellCoordinate c(4,5,6);
    std::cout << c << "\n";

    int i=c.i,j=c.j,k=c.k;
    std::cout << i << ", " << j << ", " << k;
    return 0;
}
```

Reply:
^ It looks pretty, ngl...

I feel like `cell_{i,j,k}` are useful as `private` attributes, since we don't plan to edit them and this stops us from doing that accidentally when looping/incrementing/interpolating over Yee cells. Maybe I'm being overly cautious though.

I also had the impression that somewhere in `iterator.cpp`, we need to acquire the spatial coordinates of the field values associated to each Yee cell, in which case we'd have a method on this class for doing that. But this might have already been eaten up in the refactor/ a figment of my imagination.